_posts/2019-08-23-gsoc_LibrecoresCI_Report.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: post
+title: "GSoC Project Report: Continuous Integration for Hardware Projects on LibreCores CI"
+description: ""
+category:
+tags: ['GSoC', 'LibrecoresCI' ,'FOSSi-Foundation']
+author: Nancy Chauhan
+---
+
+Librecores provides a platform to share projects and ideas, in the area of free and open-source digital hardware design. Librecores CI is a service to provide CI for hardware projects hosted on Librecores to improve user experience and reliability.
```

Comment:
Try to add links as you introduce new topics.

i.e.
[Librecores](https://www.librecores.org) provides a platform to share projects and ideas, in the area of free and open-source digital hardware design. [Librecores CI](https://www.librecores.org/static/librecores-ci) is a service to provide CI for hardware projects hosted on Librecores to improve user experience and reliability.

The more links the better.